cli/src/cli_util.rs

```diff
@@ -2311,6 +2311,14 @@ pub fn parse_args(
     Ok((matches, args))
 }
 
+pub fn format_template<T>(arg: &T, template_language: &dyn Template<T>) -> String {
+    let mut output = vec![];
+    template_language
+        .format(arg, &mut PlainTextFormatter::new(&mut output))
+        .expect("write() to PlainTextFormatter should never fail");
+    String::from_utf8(output).expect("template output should be utf-8 bytes")
+}
```

Comment:
You'll probably need to use `ui.new_formatter(buffer)` to get colorized output.

Reply:
Done. Though speaking of colors. Is there any support for users to specify colors in templates, or is it just through labels that gets applied when template functions/properties are used?

Reply:
Only labels, but there's `label()` template function to apply whatever you want.

Reply:
Ah, neat. I totally overlooked that, tested and coloring does work.